exercises/concept/high-score-board/.meta/design.md

```diff
@@ -0,0 +1,94 @@
+# Design
+
+## Learning objectives
+
+- What are objects - for now describe them as maps/dictionaries that hold key-value pairs, more advanced things will be taught later
+- Objects are the core building blocks in JavaScript
```

Comment:
To me their importance didn't come across from the exercise docs. Perhaps ask others for input on this, though, before adding more info that might just end up being redundant.

Reply:
I removed this line. I was hinting at the common saying "everything is an object" in JavaScript but I found out that this is partly a misconception when I was looking into it in more detail. :see_no_evil: